components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java

```diff
@@ -414,7 +433,13 @@ public OAuthAppDO getAppInformation(String consumerKey, int tenantId) throws
                             authenticatedUser.setUserName(rSet.getString(2));
                             oauthApp.setApplicationName(rSet.getString(3));
                             oauthApp.setOauthVersion(rSet.getString(4));
+
```

Comment:
redundant blank line

Reply:
Added it to improve readability by separating the callback URL resolving logic.